cmd/triggerrun/main.go

```diff
@@ -20,7 +20,8 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/tektoncd/triggers/cmd/triggerRun/cmd"
+	"github.com/tektoncd/triggers/cmd/triggerrun/cmd"
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
```

Comment:
```suggestion
	
```
Can we remove this import as its not used

Reply:
We actually need this one. Its contains auth plugins needed to setup the kubeclients for various k8s providers. See https://github.com/kubernetes/client-go/blob/master/examples/out-of-cluster-client-configuration/main.go#L33

e.g. https://github.com/kubernetes/client-go/blob/master/examples/out-of-cluster-client-configuration/main.go#L33

It is an interesting pattern, I'll admit 😛